romeo/asset-token.json

```diff
@@ -0,0 +1,5 @@
+{
+    "name":	"sBTC Romeo Token",
```

Comment:
Add these properties to adhere to sip 16:
```
"sip": 16,
"decimals": 8,
"symbol": "sBTC"
```

Reply:
Copy, added the three (3) sip 16 properties^